protos/hub/hub.proto

```diff
@@ -85,7 +85,7 @@ message Host {
     (gogoproto.nullable) = false,
     (gogoproto.stdtime) = true
   ];
-  string node_ip = 4;
+  string node_ip = 4 [(gogoproto.jsontag) = "-"];
```

Comment:
I think we can remove this.

Reply:
I had a chat with @laipogo on this, we cannot because of proto compatibility, I added a follow up to do a deep removal of node_ip *after* everything is deployed.

Reply:
oh, nvm, this is hub.

Reply:
done.
and i also add some `timeout api  fix` in the pr.